lib/class-wp-rest-global-styles-controller-gutenberg.php

```diff
@@ -686,9 +701,25 @@ public function get_theme_items( $request ) {
 			);
 		}
 
+		$response   = array();
 		$variations = WP_Theme_JSON_Resolver_Gutenberg::get_style_variations();
 
-		return rest_ensure_response( $variations );
+		// Add resolved theme asset links.
+		foreach ( $variations as $variation ) {
+			$variation_theme_json = new WP_Theme_JSON_Gutenberg( $variation );
+			$resolved_theme_uris  = WP_Theme_JSON_Resolver_Gutenberg::get_resolved_theme_uris( $variation_theme_json );
+			$data                 = rest_ensure_response( $variation );
+			if ( ! empty( $resolved_theme_uris ) ) {
+				$data->add_links(
+					array(
+						'https://api.w.org/theme-file-uris' => $resolved_theme_uris,
+					)
+				);
+			}
+			$response[] = $this->prepare_response_for_collection( $data );
```

Comment:
Following up on the Slack conversation @TimothyBJacobs 

Here's where I'm adding the links object to the collection. Just in case I've missed something 🤞🏻

Reply:
The rel name should be singular. But besides that looks good to me!

Reply:
I'd also remove the uri suffix. All links are uris.

Reply:
Will do! Thanks a lot 🙇🏻